lib/datasets/mapfree.py

```diff
@@ -314,10 +377,14 @@ def __init__(self, cfg, mode, transforms=None):
         assert isinstance(cfg.DATASET.QUERY_FRAME_COUNT, int)
 
         if cfg.DATASET.QUERY_FRAME_COUNT == 1:
+            if 'multi' in cfg.MODEL.lower():
+                raise ValueError(f"Model {cfg.MODEL} is not compatible with a single frame dataset!")
             sample_factor = {'train': 1, 'val': 5, 'test': 5}[mode]
             sample_offset = 0
             SceneClass = MapFreeScene
         else:
+            if 'multi' not in cfg.MODEL.lower():
```

Comment:
If we want to keep this, we should say in the README that multi-frame methods expect to contain the 'multi' string in the name. Minor detail, but people might not know why it is failing.

Reply:
ok, I made it a warning instead.